README.md

```diff
@@ -25,11 +25,12 @@ are best understood by tax attorneys, corporate lawyers, and other professionals
 Decisions related to [negotiating an offer](#offers-and-negotiations) and
 [exercising stock options](#stock-option-scenarios), in particular, can have **major financial
 consequences**. Because the value of employee equity is determined by the fate of the
-company, when a company does not succeed, an employee’s equity may be
-[worth nothing](#what-is-private-stock-worth), and the costs of exercising may not be
-recouped. Even when a company does succeed, an employee may suffer
-[catastrophic tax pitfalls](#tax-dangers) because they didn’t anticipate the tax consequences
-of their decisions.
+company, an employee’s equity may be [illiquid](#what-is-private-stock-worth) for a long time
+or ultimately [worth nothing](#growth-and-risk), and taxes and the costs of exercise, if they
+apply, may not be recouped.
+Even when a company is doing well, an employee may suffer
+[catastrophic tax pitfalls](#tax-dangers) because they didn’t fully anticipate the tax
```

Comment:
Elide "fully" (just for economy of words, that word doesn't add anything to the meaning of "anticipate" IMO). 

Removing "consequences of their decisions" connects the consequences directly to a company doing well, e.g. that there will always be tax consequences if a company does well. That could be fine, just want to make sure that's the meaning you want!